charts/rancher-monitoring/v0.3.1/Chart.yaml

```diff
@@ -9,7 +9,7 @@ sources:
 - https://github.com/coreos/prometheus-operator
 version: 0.3.1
 appVersion: 0.3.1
-kubeVersion: ">=1.16.0-0"
+kubeVersion: "< 1.22.0-0"
```

Comment:
Shouldn't this be `<1.22.0-0  >=1.16.0-0`. Why remove the 1.16.

Reply:
https://github.com/rancher/system-charts/pull/558#discussion_r770964590

Reply:
Can we use a `1.16.0-0 - 1.21.99` format instead?

Reply:
My test shows ` "1.16.0-0 - 1.21.99-0"` does not work on rancher v2.6.3-rc5 setup, so we need to keep  `"< 1.22.0-0"`

Reply:
As long as you aren't worried about this going out on <1.16 clusters. I know they aren't supported anymore.